db/migrate/20221121165352_add_type_field_to_events_reminders.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class AddTypeFieldToEventsReminders < ActiveRecord::Migration[7.0]
+  def up
+    DiscoursePostEvent::Event.where.not(reminders: nil).find_each do |event|
```

Comment:
As a rule of thumb, we don't use AR for migrations. We need to rewrite this query using `mini_sql`. See [here](https://github.com/discourse/discourse/blob/main/plugins/chat/db/post_migrate/20221117142910_delete_orphaned_channels.rb) for an example.